regexes/client/hints/browsers.yml

```diff
@@ -68,3 +68,19 @@
 'com.belva.safe.browser': 'Belva Browser'
 'com.youcare.browser': 'YouCare'
 'org.lilo.mobile.android2020': 'Lilo'
+'com.opera.cryptobrowser': 'Opera Crypto'
+'AlohaBrowser': 'Aloha Browser'
+'mark.via': 'Via'
+'com.xpp.floatbrowser': 'Float Browser'
+'com.kiddoware.kidsafebrowser': 'Kids Safe Browser'
+'com.hideitpro.vbrowser': 'vBrowser'
+'com.cgbrowser.rn': 'CG Browser'
+'com.azka.browser.anti.blokir': 'Azka Browser'
+'com.micromaxinfo.browser': 'Mmx Browser'
+'com.zeesitech.bitchutebrowser': 'Bitchute Browser'
+'nova.all.video.downloader': 'Video Downloader Pro'
+'tukidev.pronhubbrowser.tanpavpn': 'PronHub Browser'
+'com.hld.anzenbokusucal': 'Calculator Photo Vault'
+'com.hld.anzenbokusufake': 'Calculator Hide Photos'
+'com.hld.anzenbokusu': 'Sgallery'
```

Comment:
and although there is a browser here, I suggest moving it  mobile app type
https://apkpure.com/calculator-hide-photos/com.hld.anzenbokusufake